Formula/c/cargo-llvm-cov.rb

```diff
@@ -19,15 +19,10 @@ class CargoLlvmCov < Formula
 
   depends_on "rust" => :build
   depends_on "rustup-init" => :test
-  depends_on "llvm"
 
   def install
     system "tar", "--strip-components", "1", "-xzvf", "cargo-llvm-cov-#{version}.crate" unless build.head?
-    system "cargo", "install", *std_cargo_args(root: libexec)
-    llvm_bin = Formula["llvm"].opt_bin
-    (bin/"cargo-llvm-cov").write_env_script libexec/"bin/cargo-llvm-cov",
-      LLVM_COV:      llvm_bin/"llvm-cov",
-      LLVM_PROFDATA: llvm_bin/"llvm-profdata"
+    system "cargo", "install", *std_cargo_args(path: ".")
```

Comment:
`path: "."` is already the default so we can omit that:

```suggestion
    system "cargo", "install", *std_cargo_args
```